iac-extensions/arm/src/main/java/org/sonar/iac/arm/parser/bicep/TreeFactory.java

```diff
@@ -534,4 +525,17 @@ public BinaryExpression binaryExpression(Expression expression, List<Tuple<Synta
   public TernaryExpression ternaryExpression(Expression condition, SyntaxToken query, Expression ifTrueExpression, SyntaxToken colon, Expression elseExpression) {
     return new TernaryExpressionImpl(condition, query, ifTrueExpression, colon, elseExpression);
   }
+
+  private static <T extends ArmTree> SeparatedList<T, SyntaxToken> toSeparatedList(Optional<List<Tuple<Optional<SyntaxToken>, T>>> elements) {
+    SeparatedList<T, SyntaxToken> arrayContent = emptySeparatedList();
+    if (elements.isPresent()) {
+      // replace Optional<SyntaxToken> by SyntaxToken or null
+      List<Tuple<SyntaxToken, T>> elementsWithNullSeparators = elements.get().stream()
+        .map(tuple -> new Tuple<>(tuple.first().orNull(), tuple.second()))
+        .collect(Collectors.toList());
+      var firstElement = elementsWithNullSeparators.remove(0).second();
+      arrayContent = separatedList(firstElement, elementsWithNullSeparators);
```

Comment:
I think it should not be called `arrayContent`anymore since it is now also used by object expressions